k9mail/src/main/java/com/fsck/k9/preferences/SettingsImporter.java

```diff
@@ -133,7 +134,13 @@ public static ImportContents getImportStreamContents(InputStream inputStream)
             // will not be null.
             if (imported.accounts != null) {
                 for (ImportedAccount account : imported.accounts.values()) {
-                    accounts.add(new AccountDescription(account.name, account.uuid));
+                    //Show a alternative text in the case the description is empty
+                    String name = account.name;
+                    if (TextUtils.isEmpty(name) && account.identities != null && account.identities.size() > 0){
```

Comment:
Hm. is that worth it for `ImportedAccount `? It's maybe more useful to just create a method in `Account `to replace things like this:
https://github.com/k9mail/k-9/blob/master/k9mail/src/main/java/com/fsck/k9/activity/Accounts.java#L1761.